docs/installation.rst

```diff
@@ -57,9 +57,13 @@ Download
 
 #. Change directories to repo
 
-#. Install dependencies with ``npm``
+#. Install `yarn <https://yarnpkg.com/en/docs/install#alternatives-stable>`_
 
-   - ``npm install``
+   - ``curl -o- -L https://yarnpkg.com/install.sh | bash``
```

Comment:
Personally I'd prefer to give instructions other than piping a shell script to Bash. Is this what upstream recommends?

Reply:
I can link to the install docs, would you prefer that?